ragstack-e2e-tests/e2e_tests/llama_index/test_compatibility_rag.py

```diff
@@ -27,37 +34,120 @@
     get_astra_ref,
     delete_all_astra_collections,
     delete_astra_collection,
+    AstraRef,
 )
+from llama_index.vector_stores.types import VectorStore, VectorStoreQuery
+from sqlalchemy import Float
+from vertexai.vision_models import MultiModalEmbeddingModel, Image
+
+
+class VectorStoreWrapper:
```

Comment:
I would call it `VectoreStore` directly.
It will not collide with LI's VectorStore if we remove `as_vector_store `